### PR TITLE
[MO] - [system test] - fixes of STs

### PR DESCRIPTION
**Pull request by Seequick1**
### Type of change

- Bugfix
- Enhancement / new feature

### Description

This PR adding fixes some few test cases. 

```
io.strimzi.systemtest.tracing.TracingST.testConnectService
io.strimzi.systemtest.RollingUpdateST.testRecoveryDuringZookeeperRollingUpdate
io.strimzi.systemtest.specific.SpecificST.testDeployUnsupportedKafka
io.strimzi.systemtest.RollingUpdateST.testRecoveryDuringKafkaRollingUpdate
```

### Checklist

- [x] Write tests
- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java [hunk 1]
```diff
@@ -1057,4 +1059,18 @@ protected void checkReadinessLivenessProbe(String podNamePrefix, String containe
             fail("Pod with prefix " + podNamePrefix + " in name, not found");
         }
     }
+
+    protected void verifyCRDStatusCondition(Condition condition, String status, String type) {
```
Comment (scholzj) [inline systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java]: Should this be just `verifyCRStatusCondition` and `verifyCRStatusCondition`?

File: systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java [hunk 2]
```diff
@@ -1019,4 +1019,28 @@ public static void waitUntilPVCAnnotationChange(Map<String, String> newAnnotatio
             });
         LOGGER.info("PVC annotation has changed {}", newAnnotation.toString());
     }
-}
+
+    public static void waitUntilKafkaUserStatusConditionIsPresent(String userName) {
+        LOGGER.info("Waiting till kafka user name:{} is created in CRDs", userName);
```
Comment (scholzj) [inline systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java]: Again ... CR versus CRD?

File: systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java (hunk 1 above)
Comment (Seequick1) [reply systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java]: yeah that's correct :)

File: systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java [hunk 3]
```diff
@@ -122,21 +120,16 @@ void testDeployUnsupportedKafka() {
...
+        Condition condition = testMethodResources().kafka().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus().getConditions().get(0);
 
-    // This should be moved to StUtils in the futures
-    void assertKafkaStatus(String status, String message) {
-        KafkaStatus kafkaStatus = testMethodResources().kafka().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getStatus();
-        assertThat("Kafka Kafka cluster status should be " + status, kafkaStatus.getConditions().get(0).getType(), is(status));
-        assertThat("Kafka cluster status message should be different", kafkaStatus.getConditions().get(0).getMessage(), containsString(message));
+        verifyCRStatusCondition(
+                condition,
+                "Version 6.6.6 is not supported. Supported versions are: 2.2.1, 2.3.0.",
```
Comment (scholzj) [inline systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificST.java]: This is very fragile. Would it be better to just check that _"Version 6.6.6 is not supported_ is part of that message?

**Pull request merged**